app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_and_belongs_to_many :review_requests
+  has_many :owned_review_requests, class_name: "ReviewRequest", foreign_key: "owner_id", dependent: :destroy, inverse_of: :owner
+
+  def self.create_or_find (json)
```

Comment:
https://github.com/rootstrap/tech-guides/tree/master/ruby#spaces-braces

Reply:
this makes me think.. is rubocop correctly configured? Why didn't it catch any of these syntax errors?